include/modules/hyprland/workspaces.hpp

```diff
@@ -9,27 +9,30 @@
 
 namespace waybar::modules::hyprland {
 
-struct WorkspaceDto {
-  int id;
-
-  static WorkspaceDto parse(const Json::Value& value);
-};
-
 class Workspace {
  public:
-  Workspace(int id);
-  Workspace(WorkspaceDto dto);
-  int id() const { return id_; };
-  int active() const { return active_; };
+  Workspace(const Json::Value& value);
   std::string& select_icon(std::map<std::string, std::string>& icons_map);
-  void set_active(bool value = true) { active_ = value; };
   Gtk::Button& button() { return button_; };
 
+  int id() const { return id_; };
+  std::string name() const { return name_; };
+  std::string monitor() const { return monitor_; };
+  int active() const { return active_; };
+  bool is_special() const { return is_special_; };
+
+  auto handle_clicked(GdkEventButton* bt) -> bool;
+  void set_active(bool value) { active_ = value; };
+
   void update(const std::string& format, const std::string& icon);
 
  private:
   int id_;
+  std::string name_;
+  std::string monitor_;
```

Comment:
I can suggest name it `output`, as in wayland ecosystem.

Reply:
sure, was using hyprland's name